storage/database.go

```diff
@@ -375,7 +363,8 @@ func (d *db) getOwnedNamespaces() []databaseNamespace {
 }
 
 func (d *db) reportLoop() {
-	t := time.Tick(time.Second) // TODO: Make this an option, what is a good default
+	interval := d.opts.GetInstrumentOptions().GetGaugeInterval()
```

Comment:
Perhaps call it `GetReportInterval` instead in case we want to add more metrics to the `reportLoop` method?